raspberry/bzzz/sensors/gnss.py

```diff
@@ -169,52 +275,114 @@ def all_gnss_data(self):
+        relative to the base station
         """
         with self.__lock:
             return self.__data_processor.process(self.__values_cache[:, 0], 
                                                  cursor=self.__cursor)
 
     @property
-    def longitude(self):
+    def relative_east(self):
         """
-        Returns Longitude position in decimal
+        The distance of the quadcopter in meters in the east direction relative
+        to the base station
         """
         with self.__lock:
             return self.__data_processor.process(self.__values_cache[:, 1], 
                                                  cursor=self.__cursor)
+        
+    @property
+    def relative_down(self):
+        """
+        The distance of the quadcopter in meters in the down direction relative
+        to the base station
+        """
+        with self.__lock:
+            return self.__data_processor.process(self.__values_cache[:, 2], 
+                                                 cursor=self.__cursor)
 
     @property
     def altitude(self):
         """
-        Returns the Altitude position
+        Returns the Altitude of the quadcopter in meters based of the relative
+        Down position of the quadcopter compared to the base station.
         """
         with self.__lock:
-            current_altitude = self.__data_processor.process(self.__values_cache[:, 2], cursor=self.__cursor)
-            # Check if __average_altitude is not None and subtract it from current altitude
+            current_altitude = - self.__data_processor.process(
+                self.__values_cache[:, 2], cursor=self.__cursor)
+            # Check if __average_altitude is not None and subtract it from
+            # current altitude
             if self.__average_altitude is not None:
                 return current_altitude - self.__average_altitude
             else:
                 return current_altitude
-            
+
+    @property
+    def position_longitude(self):
+        """
+        Returns Longitude position in decimal degrees
+        """
+        with self.__lock:
+            return self.__data_processor.process(self.__values_cache[:, 5], 
+                                                 cursor=self.__cursor)
+        
+    @property
+    def position_latitude(self):
+        """
+        Returns Latitude position in decimal degrees
+        """
+        with self.__lock:
+            return self.__data_processor.process(self.__values_cache[:, 6], 
+                                                 cursor=self.__cursor)
+        
+    @property
+    def position_altitude(self):
+        """
+        Returns the Altitude position
+        """
+        with self.__lock:
+            return self.__data_processor.process(self.__values_cache[:, 7], 
+                                                 cursor=self.__cursor)
+        
+    @property
+    def rtk_status(self):
+        """
+        Returns the RTK status
```

Comment:
Can you write a comment on what this returns? It's not clear that it returns `True`/`False` values only.

Reply:
done